src/CaptureClient/CaptureEventProcessor.cpp

```diff
@@ -553,7 +553,10 @@ void CaptureEventProcessorForListener::ProcessThreadNamesSnapshot(
 void CaptureEventProcessorForListener::ProcessThreadStateSlice(
     const ThreadStateSlice& thread_state_slice) {
   ThreadStateSliceInfo slice_info{
-      thread_state_slice.tid(), thread_state_slice.thread_state(),
+      thread_state_slice.tid(),
+      thread_state_slice.woker_tid(),
+      thread_state_slice.woker_pid(),
```

Comment:
As discussed in the other comment. `woker` is not proper English. I would rename this to `was_blocked_by_tid` and `was_blocked_by_pid`. (Yeah I know, this needs to be changed in many places. But a project-wide search and replace should work fairly well.)